piet/src/color.rs

```diff
@@ -100,6 +100,8 @@ impl Color {
     /// which is perhaps not ideal (the clipping might change the hue). See
     /// https://github.com/d3/d3-color/issues/33 for discussion.
     #[allow(non_snake_case)]
+    #[allow(clippy::many_single_char_names)]
```

Comment:
And people wonder why Rust isn't considered a first choice for scientific code :)